span_test.go

```diff
@@ -522,7 +522,7 @@ func TestCompressSpanSameKind(t *testing.T) {
 	}
 
 	t.Run("DefaultThreshold", func(t *testing.T) {
-		// With the default threshold the composite count will be 5.
+		// By default same kind compression is disabled thus count will be 7.
```

Comment:
The subtest name is a little confusing now. Perhaps rename to `DefaultDisabled`?

Reply:
Sounds better, will update